helix-term/src/ui/editor.rs

```diff
@@ -1233,6 +1234,13 @@ impl Component for EditorView {
             }
 
             Event::Mouse(event) => self.handle_mouse_event(event, &mut cx),
+            Event::Focus(FocusEventKind::Gained) => EventResult::Ignored(None),
+            Event::Focus(FocusEventKind::Lost) => {
+                if let Err(e) = commands::typed::write_all(context, &[], PromptEvent::Validate) {
```

Comment:
It seems like I should call `MappableCommand.execute` instead of recreating a bit of its implementation here and making `write_all` public. I didn't see a good way to do that for `TypableCommand` like there is for static commands. Am I missing something?